runsc/console/pty_darwin.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build darwin
+// +build darwin
+
+package console
+
+import "golang.org/x/sys/unix"
+
+func IsPty(fd uintptr) bool {
```

Comment:
Needs doc comment, otherwise go vet will complain:  `IsPty returns true if FD is a PTY`.

Reply:
Comment added (inside `pty_linux.go` as I removed `pty_darwin.go`).
https://github.com/google/gvisor/blob/d5e1ed3cd98d125c729d62561aab4ecd6bc9e4b5/runsc/console/pty_linux.go#L20-L23